spec/rubocop/cop/rspec/variable_definition_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::VariableDefinition, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'when EnforcedStyle is `symbols`' do
+    let(:cop_config) { { 'EnforcedStyle' => 'symbols' } }
+
+    it 'registers an offense for string name' do
+      expect_offense(<<~RUBY)
+        let("user_name") { 'Adam' }
+            ^^^^^^^^^^^ Use symbols for variable names.
+      RUBY
+    end
+
+    it 'registers an offense for interpolated string' do
+      expect_offense(<<~'RUBY')
+        let("user-#{id}") { 'Adam' }
```

Comment:
Sorry for adding a comment to a long closed PR, but… what’s the benefit of converting this interpolated string into a symbol? Sure, we’re consistently passing the same _type_ to our `let` calls, but it feels like double work to not just pass the interpolated string.

Reply:
Good point.

It was originally brought up here https://github.com/rubocop-hq/rubocop-rspec/pull/907#discussion_r418586020 by @Darhazer 

Interpolated variable names seem to be an exception (though I remember someone mentioned it was used in some specific specs).

I'm all for adding an exception for interpolated var names (just not up for it due to my overflown queue of things to fix).